tests/integration/targets/utils_cli_parse/tasks/main.yaml

```diff
@@ -8,6 +8,13 @@
   vars:
     parser: "({{ inventory_hostname }}/argspec)"
 
+- name: Include cli_parse.yaml
```

Comment:
Minor comment: Can we improve the naming of this task and be more specific to _what_ we're testing here?